src/test/resources/net/sf/jabref/importer/fileformat/OvidImporterTestBib6.bib

```diff
@@ -1,6 +1,6 @@
 @Misc{,
   Title                    = {Correct positioning of percutaneous iliosacral screws with computer-navigated versus fluoroscopically guided surgery in traumatic pelvic ring fractures},
-  Author                   = {Verbeek, Jan MD and Hermans, Erik MD and van Vugt, Arie MD, PhD and Frolke, Jan Paul MD, PhD},
```

Comment:
Yeah... you are right it is a bit annoying. But I think the majority of author names does not contain the degree (MD, PHD, ...) of the author or set it up properly as the Jr part.